modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1424,8 +1424,7 @@ def rebalance_partitions(cls, partitions):
                     partition_size = ideal_partition_size
                 new_partitions.append(
                     cls.column_partitions(
-                        (partitions[start : stop + 1]),
-                        full_axis=partition_size == total_rows,
+                        (partitions[start : stop + 1]), full_axis=False
```

Comment:
Can we add a comment that we're setting it to False on purpose since setting it to True will cause the partition to be split again after the next apply, which doesn't make sense, since we just concatenated it?

Reply:
I added a comment that makes more sense to me.